bower.json

```diff
@@ -32,15 +32,14 @@
     "url": "git://github.com/patternfly/angular-patternfly.git"
   },
   "dependencies": {
-    "angular": "1.2.25 - 1.3.*",
-    "patternfly": "~2.0.0",
-    "c3": "~0.4.10"
```

Comment:
As an FYI there is an effort going on to make the patternfly reference dependencies true bower dependencies so we might need to update this at a later date but knowing what we know right now I think this is the way to go.